### PR TITLE
Added datadogMetricsService and fix datadogNamespace key in triggerAu…

### PR DESCRIPTION
**Pull request by alvarobacelar**
Included `datadogMetricsService` and fixed `datadogNamespace` key in triggerAuthenticatio specs. When I have been Tried this configurations I got errors. After I fix this keys, provide in this PR, works well. 

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

**Comment by zroubalik** [comment]
Thanks, duplicate of: https://github.com/kedacore/keda-docs/pull/1451/

**Comment by alvarobacelar** [comment]
Hello @zroubalik , but this PR fix the namespace as well. 

https://github.com/kedacore/keda-docs/pull/1469/files#diff-015e70f7405a2745734c1d02b964c2a3990ff85783e2bbf9f31a12037c115552R144

**Comment by zroubalik** [comment]
@alvarobacelar thanks, haven't notice that. Could you please rebase this PR ?

**Comment by alvarobacelar** [comment]
@zroubalik I've been done.

File: content/docs/2.15/scalers/datadog.md [hunk 1]
```diff
@@ -141,7 +141,10 @@ spec:
       key: token
     - parameter: datadogNamespace
       name: datadog-config
-      key: namespace
+      key: datadogNamespace
+    - parameter: datadogMetricsService
+      name: datadog-config
+      key: datadogMetricsService
```
Comment (zroubalik) [inline content/docs/2.15/scalers/datadog.md:147]: Please don't add this, it is already specifeid below

Comment (alvarobacelar) [reply content/docs/2.15/scalers/datadog.md]: I've been done.

Comment (alvarobacelar) [reply content/docs/2.15/scalers/datadog.md]: @zroubalik something missing in this PR?

**Comment by alvarobacelar** [comment]
Hi @zroubalik. is it everything ok with this PR?

**Comment by alvarobacelar** [comment]
Hello, @JorTurFer. 
I think this PR it's ok. 
Could you please review it?

**Comment by alvarobacelar** [comment]
@zroubalik what happened with this PR? 
Is there something wrong? Please let me know if there’s anything wrong.

**Review by zroubalik — APPROVED** [review]
LGTM, thanks @alvarobacelar !

Sorry for the delay, a lot of stuff on my backlog :)

**Pull request merged**